components/TokenSelectionModal.tsx

```diff
@@ -82,8 +82,9 @@ const TokenSelectionModal = (props: TokenSelectionModalProps) => {
   const { tokenList } = useToken();
   const [searchTerm, setSearchTerm] = useState("");
   const options = {
-    includeScore: false,
-    keys: ["address", "name"],
+    includeScore: true,
+    keys: ["address", "name", "symbol"],
+    threshold: 0.2,
```

Comment:
Just curious how you chose this threshold? (0.2 is totally fine with me, just wondering if it's based on testing, past usage of fuse, etc)

Reply:
It was mainly testing and specifically the use cases you mentioned above, I haven't used fuse before, but it seemed like the best library for the bundle hit. .1 is probably better because it is more restrictive after taking another look at the docs.

https://fusejs.io/concepts/scoring-theory.html#fuzziness-score

Reply:
I've used fuse in the past, it is a nice lib and very configurable. But yea 0.1 or 0.2 both work for me, we can always change it easily in the future